sql/2021-08-30-add-test-user.sql

```diff
@@ -0,0 +1,6 @@
+
+
+insert into User(username, password, isAdmin)
+values
+('username@kainos.com', '$2a$04$cKdU1pCkScsJrT9lCdD/A.44crtlEvwufG5tO/oFEBviwpEwzrNy.', 0), --password
+('admin@kainos.com', '$2a$04$qcm3tUsOfYabxN0Dq61JuOYUEFPhFzYw/dqqVYdXlOxCM91I3vcj2', 1); --admin_password
```

Comment:
Is it a good idea to have this committed?

Reply:
I mean the passwords are hashed so it's secure?

Reply:
Just wanted to check